### ISSUE TITLE
When connected to the external monitor, the ICONS appear confused

### ISSUE THREAD

**Issue by XHXIAIEIN** [Scirra/Construct-3-bugs#3479]
## Problem description

I was working normally on my MacBook Pro, But after I connected to the external display and browser window( including the Chrome APP) moved to the display. May be the reason for the different screen resolutions. All the event condition icons are displayed in a messy pattern.

[Imgur](https://i.imgur.com/ErEUZgJ.png)
The original event icon in the event sheets has no affected, but new add event action icon will change.

Of course, the problem was solved after the restart Chrome. 
Even if save the file at this time, the messy icon will not appear after the restart.
So here is just a feedback.

## More details

**Affected browsers/platforms:**  Chrome, macOS
**First affected release:**  r175

## System details

<details><summary>View details</summary>

Platform information
Browser: Chrome
Browser version: 78.0.3904.97
Browser engine: Chromium
Browser architecture: (unknown)
Context: browser
Operating system: Mac OS X
Operating system version: 10.15_1
Operating system architecture: (unknown)
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 8
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.97 Safari/537.36
C3 release: r175 (beta)
Language setting: en-US

Local storage
Storage quota (approx): 67 gb
Storage usage (approx): 191 mb (0.3%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: Intel Inc.
Renderer: Intel(R) Iris(TM) Plus Graphics 655
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 255.875
Extensions:

EXT_color_buffer_float
EXT_disjoint_timer_query_webgl2
EXT_float_blend
EXT_texture_filter_anisotropic
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

<details><summary>View details</summary>

Platform information
Browser: Chrome
Browser version: 78.0.3904.97
Browser engine: Chromium
Browser architecture: (unknown)
Context: browser
Operating system: Mac OS X
Operating system version: 10.15_1
Operating system architecture: (unknown)
Device type: desktop
Device pixel ratio: 1
Logical CPU cores: 8
Approx. device memory: 8 GB
User agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.97 Safari/537.36
C3 release: r175 (beta)
Language setting: en-US

Local storage
Storage quota (approx): 67 gb
Storage usage (approx): 191 mb (0.3%)
Persistant storage: Yes

Browser support notes
This list contains missing features that are not required, but could improve performance or user experience if supported.

Nothing is missing. Everything is OK!
WebGL information
Version string: WebGL 2.0 (OpenGL ES 3.0 Chromium)
Numeric version: 2
Supports NPOT textures: yes
Supports GPU profiling: yes
Supports highp precision: yes
Vendor: Intel Inc.
Renderer: Intel(R) Iris(TM) Plus Graphics 655
Major performance caveat: no
Maximum texture size: 16384
Point size range: 1 to 255.875
Extensions:

EXT_color_buffer_float
EXT_disjoint_timer_query_webgl2
EXT_float_blend
EXT_texture_filter_anisotropic
OES_texture_float_linear
WEBGL_compressed_texture_s3tc
WEBGL_compressed_texture_s3tc_srgb
WEBGL_debug_renderer_info
WEBGL_debug_shaders
WEBGL_lose_context
Audio information
System sample rate: 48000 Hz
Output channels: 2
Output interpretation: speakers
Supported decode formats:

Ogg Opus (audio/ogg; codecs=opus)
WebM Opus (audio/webm; codecs=opus)
Ogg Vorbis (audio/ogg; codecs=vorbis)
WebM Vorbis (audio/webm; codecs=vorbis)
MPEG-4 AAC (audio/mp4; codecs=mp4a.40.5)
MP3 (audio/mpeg)
FLAC (audio/flac)
PCM WAV (audio/wav; codecs=1)
Supported encode formats:

WebM Opus (audio/webm; codecs=opus)
Video information
Supported decode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)
Ogg Theora (video/ogg; codecs=theora)
H.264 (video/mp4; codecs=avc1.42E01E)
Supported encode formats:

WebM VP9 (video/webm; codecs=vp9)
WebM VP8 (video/webm; codecs=vp8)

</details>

**Comment by AshleyScirra** [comment]
I can't think how any of our code would produce this result. It looks like a browser bug or maybe a GPU driver issue. Can you reproduce the issue in Firefox and Safari as well?

**Comment by XHXIAIEIN** [comment]
Yes, it can still be reproduced in Safari.
When I connected the display  and moved the window to the display, It reappears.

I also noticed that there was a tips window this time.
[Imgur](https://imgur.com/MVb2yY2)

And there was new finding.
If I move back to the built-in Retina display, it's back to normal.

I think it's just that the two display don't have the same resolution, causing problems with icon scaling.
